tests/Doctrine/Tests/DBAL/Functional/Driver/OCI8/StatementTest.php

```diff
@@ -50,6 +67,11 @@ public static function queryConversionProvider() : iterable
                 [1],
                 ['COL1' => 1],
             ],
+            'named' => [
```

Comment:
Yeah, this seems to be indeed a missing case. Would it make sense to rename the previous data point from "simple" to "positional" to better reflect the difference between them?